frontend/libs/studio-components/src/components/StudioIconTextfield/StudioIconTextfield.tsx

```diff
@@ -10,17 +10,20 @@ export type StudioIconTextfieldProps = {
 
 export const StudioIconTextfield = forwardRef<HTMLDivElement, StudioIconTextfieldProps>(
   (
-    { icon, className: givenClassName, ...rest }: StudioIconTextfieldProps,
+    { icon, className: givenClassName, children, ...rest }: StudioIconTextfieldProps,
     ref,
   ): React.ReactElement => {
     const className = cn(givenClassName, classes.container);
     return (
-      <div className={className} ref={ref}>
-        <div aria-hidden className={classes.prefixIcon}>
-          {icon}
+      <>
+        <div className={className} ref={ref}>
+          <div aria-hidden className={classes.prefixIcon}>
+            {icon}
+          </div>
+          <StudioTextfield {...rest} className={classes.textfield} />
         </div>
-        <StudioTextfield {...rest} className={classes.textfield} />
-      </div>
+        {children}
```

Comment:
It's for displaying an infoAlert under the componentId textField when it is in editMode. If it is placed directly underneath the `<StudioTextField/>` it will look like this:
![Skjermbilde 2024-06-18 kl  10 24 56](https://github.com/Altinn/altinn-studio/assets/71079896/a79601f7-7857-48fa-b2d5-87d315c71151)

But there might be another way to solve it that is more convinient?

Reply:
hm. based on the picture it looks like CSS issues? What about wrapping it with div? <div><textifled /><alert/></div>? 😊